trace/stats.go

```diff
@@ -127,6 +130,10 @@ func (s *PacketCounter) Update(c PacketCounts) {
 	s.total.Add(c)
 }
 
+func (s *PacketCounter) Get() PacketCounts {
+	return s.total
+}
```

Comment:
If the function right below this that does the exact same thing uses a read mutex, it's a fair bet that this one has to as well.